bot/toto.js

```diff
@@ -773,18 +255,59 @@ function handleCmdDonate(device_address){
 	return device.sendMessageToDevice(device_address, 'text',  msg);
 }
 
-exports.readSingleWallet = readSingleWallet;
-exports.readSingleAddress = readSingleAddress;
-exports.signer = signer;
-exports.isControlAddress = isControlAddress;
-exports.issueOrSelectNextMainAddress = issueOrSelectNextMainAddress;
-exports.issueNextMainAddress = issueNextMainAddress;
-exports.issueOrSelectStaticChangeAddress = issueOrSelectStaticChangeAddress;
-exports.issueChangeAddressAndSendPayment = issueChangeAddressAndSendPayment;
-exports.setupChatEventHandlers = setupChatEventHandlers;
-exports.handleText = handleText;
-exports.sendAllBytesFromAddress = sendAllBytesFromAddress;
-exports.sendAssetFromAddress = sendAssetFromAddress;
-
-if (require.main === module)
-	setupChatEventHandlers();
+function handleBuyTicket(device_address, ticket_count_text){
+
+	var device = require('byteballcore/device.js');
+	
+	var buyCmdInfo = "Buy command:";
+	buyCmdInfo += "\n[buy](command:buy): buy 1 ticket";
+	buyCmdInfo += "\n[buy n](command:buy): buy n tickets";
+
+	var ticket_count = 1;
+	
+	if(ticket_count_text && ticket_count_text != ''){
+		ticket_count = Number.parseInt(ticket_count_text);
+		if (Number.isNaN(ticket_count)) {
+			return device.sendMessageToDevice(device_address, 'text', "'"+ticket_count_text+"' is not a number. \n"+buyCmdInfo);
+		}
+	}
+
+	if (ticket_count<=0) {
```

Comment:
to be safe I would do  
if(...){....}
if else (ticket_count<=0) {..}
else { return; )//respond that invalid value, catches any other weird input
Currently would  go to issuing